frontend/src/pages/Products/PageProductChild.vue

```diff
@@ -1,85 +1,124 @@
 <template>
-	<q-layout container style="height: 100vh" view="lHh lpR lFf">
-		<q-header class="bg-white main_header">
-			<q-toolbar>
-				<div class="row">
-					<q-btn
-						flat
-						@click="drawer = !drawer"
-						round
-						color="primary"
-						icon="menu"
-					/>
-				</div>
-				<AccountSettings />
-			</q-toolbar>
-		</q-header>
-		<AjaxBar />
-
-		<q-drawer
-			side="right"
-			show-if-above
-			bordered
-			:width="250"
-			:breakpoint="500"
-			class="bg-grey-3"
-		>
-			<h6>Object Details</h6>
-			<div id="detailsContainer">
-				<ConfigPannel></ConfigPannel>
-			</div>
-		</q-drawer>
-		<Drawer v-model="drawer">
-			<template v-slot:drawerFilter v-if="isMounted">
-				<PaletteMenu id="paletteMenu" :data="paletteData"></PaletteMenu>
-			</template>
-			<template v-slot:drawerMenu>
-				<div class="q-pa-md q-gutter-sm" v-if="menu">
-					<q-tree
-						:nodes="menu"
-						node-key="label"
-						:filter="filter"
-						v-model:selected="selected"
-						default-expand-all
-					/>
-				</div>
+	<div>
+		<ProductHeader>
+			<template v-slot:productHeaderContent>
+				<section class="productDetails_navbar">
+					<div class="partOne">
```

Comment:
rename partOne and partTwo by something understandable

Reply:
ok